### PR TITLE
WINUI-2297: Slider UG update based on windows 11 UI changes

### PR DESCRIPTION
**Pull request by sivasuryanarayananv**
Slider UG output images update

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -4,4 +4,12 @@ yarn.lock
 package-lock.json
 cireports
 package.json
-gulpfile.js
\ No newline at end of file
+gulpfile.js
```
Comment (vigneshrameshsync) [inline .gitignore:7]: Whether these changes needed?

**Review by vigneshrameshsync — APPROVED** [review]
The changes are fine.

File: .gitignore (hunk 1 above)
Comment (sivasuryanarayananv) [reply .gitignore]: I have completed all the above mentioned changes sir

**Review by vigneshrameshsync — COMMENTED** [review]
The corrections are fine.

**Pull request merged**